blog.js

```diff
@@ -0,0 +1,7 @@
+
+sortArticlesByDate(blog);
+
+for (var i = 0; i < blog.articles.length; i++){
+  var fullArticle = new CompleteArticle(blog.articles[i]);
+  fullArticle.toHTML();
+}
```

Comment:
It would be best here to wrap your for loop here in a function to keep it out of the global scope.